components/proxy/src/test/java/com/hotels/styx/proxy/BackendServicesRouterTest.java

```diff
@@ -63,6 +69,15 @@
 
     private Environment environment;
 
+    private ClientEventLoopFactory factory = new PlatformAwareClientEventLoopGroupFactory("x", 0);
+    private EventLoopGroup eventLoopGroup = this.factory.newClientWorkerEventLoopGroup();
+    private Class<? extends SocketChannel> channelClass = this.factory.clientSocketChannelClass();
+
+    @AbstractAnnotationSpec.AfterAll
```

Comment:
This is a java test, not a kotlin test.
```suggestion
    @AfterClass
```